sqlelf/elf/instruction.py

```diff
@@ -63,4 +63,10 @@ def register(connection: apsw.Connection, binaries: list[lief.Binary]):
     generator.columns, generator.column_access = apsw.ext.get_column_names(
         next(generator())
     )
-    apsw.ext.make_virtual_module(connection, "elf_instructions", generator)
+    apsw.ext.make_virtual_module(connection, "raw_elf_instructions", generator)
+    connection.execute(
```

Comment:
TODO: maybe make this a flag to control whether a user wants to pay the cost?
Might be worthwhile for tiny binaries.